### ISSUE TITLE
[BUG] Extension does not use build task from tasks.json for tests

### ISSUE THREAD

**Issue by brizdotdev** [microsoft/vscode-dotnettools#198]
### Describe the Issue

In my `tasks.json` I have the build task set to not open the panel and close the terminal instead of the "Terminal will be reused by tasks, press any key to close it." message.

```json
{
      "label": "build",
      "command": "dotnet",
      "type": "process",
      "args": [
        "build",
        "${workspaceFolder}/MySolution.sln",
        "/property:GenerateFullPaths=true",
        "/consoleloggerparameters:NoSummary"
      ],
      "problemMatcher": "$msCompile",
      "presentation": {
        "echo": true,
        "reveal": "never",
        "focus": false,
        "panel": "shared",
        "showReuseMessage": false,
        "clear": true,
        "close": true,
      }
    }
```

However, when running a test from the testing view, the panel still opens and the reuse message still appears.

Is there a way to customize the build task used when running tests?

### Steps To Reproduce

1. Create a new .NET project with tests
2. Add the following to build task:
```json
      "presentation": {
        "echo": true,
        "reveal": "never",
        "focus": false,
        "panel": "shared",
        "showReuseMessage": false,
        "clear": true,
        "close": true,
      }
 ```

### Expected Behavior

When a test is run, the build task is invoked and the panel does not appear and the terminal used for the build is closed after.

### Environment Information

- VSCode: Version: 1.80.0-insider
- C# DevKit: v0.1.103
- OS: Windows 11

**Comment by ocallesp** [comment]
@ocallesp need to reproduce

**Comment by AbhitejJohn** [comment]
This is likely a little old but did we end up fixing this as part of your recent changes @peterwald ?

**Comment by peterwald** [comment]
No, the C# test explorer builds through the project system and has no knowledge of the `tasks.json` infrastructure. We don't have a way to accomplish this without unification of the C# project system and VS Code build tasks.

**Comment by florestankorp** [comment]
I've found a few similar issues:

https://github.com/microsoft/vscode-dotnettools/issues/198
https://github.com/microsoft/vscode-dotnettools/issues/807
https://github.com/microsoft/vscode-dotnettools/issues/1695
https://github.com/microsoft/vscode-dotnettools/issues/2183

I'd say let's join forces and focus on one, so we can get it prioritized on to the backlog.

PS: I also sent @timheuer a tweet (?) on BlueSky :)
